pyqg/tests/test_xarray_output.py

```diff
@@ -142,8 +148,17 @@ def test_xarray(all_models):
             for v in list(ds.keys()): 
                 assert v in expected_vars + expected_diags
 
+            for v in expected_diags:
+                if v in ds:
+                    assert 'time' in ds[v].coords
+
             for a in expected_attrs:
                 assert f"pyqg:{a}" in ds.attrs
 
             for c in expected_coords:
                 assert c in ds.coords
+
+    if len(datasets):
```

Comment:
I had to add this `if` because some of the models in `all_models` actually never get any snapshots given their current settings... this might be an issue with the existing tests!

Reply:
Thanks @asross for point this out! I've adjusted the model parameters so that they all get snapshots.

Reply:
Awesome!